### PR TITLE
client starter

### PR DESCRIPTION
**Pull request by jtoney1**
added web components(polyfill), angular, polymer, bower, and the ui/routes for each partial

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
```
Comment (ksmithut) [inline bower.json:17]: We need to have angular and angular-route in here. On that note, we need to figure out which browsers we need to support in order to know which version of angular to use. Angular 1.3 dropped support for ie8

File: client/partials/abtAppropriations.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="abtAppropriations">
```
Comment (ksmithut) [inline client/partials/abtAppropriations.html]: What are your thoughts on class naming? Most css/html standards say to use all lowercase with words separated by hyphens, and some html linters will throw an error if it's not. I'm fine with it being this way, just wanted to know your thoughts.

File: client/partials/abtAppropriations.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+    <h1>About the Appropriations Chair</h1>
+</div>
```
Comment (ksmithut) [inline client/partials/abtAppropriations.html]: Also, git doesn't like it when we don't have an empty newline at the end of a file. We might want to fix that to make git happy.

File: config/index.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ var config = convict({
    * @example
    * $ NODE_ENV=production npm start
    * @example
-   * $ node app --env production
+   * $ node client --env production
```
Comment (ksmithut) [inline config/index.js]: What was the reason for changing these from app to client?

**Comment by ksmithut** [comment]
@jtoney1 See my notes so that we can get merged in soon.

File: bower.json (hunk 1 above)
Comment (jtoney1) [inline bower.json:17]: I added those with a CDN via the index.html.  I just did the latest version I think...  If we are using Polymer (Which I would like to) I think the webcomponents.js polyfill only supports ie10+  ( I haven't really investigated this, so maybe there is a way around it)

File: client/partials/abtAppropriations.html (hunk 2 above)
Comment (jtoney1) [inline client/partials/abtAppropriations.html]: I didnt consider that, I'm fine with the dash standard.

File: config/index.js (hunk 4 above)
Comment (jtoney1) [inline config/index.js]: This was something I was toying with and didnt change it back, sorry.

File: bower.json (hunk 1 above)
Comment (jtoney1) [inline bower.json:17]: I'm fine pulling them in with bower... I just figured it was less resource hungry this way.

Comment (ksmithut) [inline bower.json:17]: Ah, I see that now. We might want to look into bundling it with our app in order to reduce the number of HTTP requests, but I think for now it should be fine. It's something that can be added later pretty easily if we decide to go that route.

Comment (ksmithut) [inline bower.json:17]: If we bundle them, we don't have to rely on a 3rd party server, but in all likelihood, their CDN servers will have much more uptime than ours. There are just enough freak stories to scare me into downloading them (like malicious code on the jQuery CDN).

But like I said, it's something pretty simple to add in later. Let's now worry about it for now.

**Pull request merged**